applications/API/forms.py

```
@@ -0,0 +1,284 @@
+        logger.error(f"Error getting cached insult data: {e}")
+        return [], "[]"
+
+
+def invalidate_insult_cache(reason: str = "manual") -> None:
+    """
+    Invalidate insult cache.
+
+    Args:
+        reason: Reason for invalidation
+    """
+    insult_choices_manager.invalidate_cache(reason)
```

Ruff (0.12.2)

Undefined name `insult_choices_manager`

(F821)